### DISCUSSION TITLE
s/i-has-settled-down-in-shanghai

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#23] [Announcements]
# s/i-has-settled-down-in-shanghai

沪漂十余年我拿到了上海户口

https://blog.ops-coffee.cn/s/i-has-settled-down-in-shanghai

**Comment by cordream** [comment]
恭喜，恭喜。。。

**Comment by AaronConlon** [comment]
Cool

**Comment by woshizhd** [comment]
恭喜恭喜。我的终点应该是回农村养老哈哈哈，对空气、环境比较在意😆

**Comment by ryan4yin** [comment]
超酷的前辈！

**Comment by iawes** [comment]
楼主很励志，吾辈楷模
